apps/lfx-one/src/app/modules/profile/linux-email/profile-linux-email.component.html

```
@@ -0,0 +1,187 @@
+<!-- Copyright The Linux Foundation and each contributor to LFX. -->
+<!-- SPDX-License-Identifier: MIT -->
+
+<div class="w-full" data-testid="profile-linux-email">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)